applications/FSIApplication/python_scripts/trilinos_partitioned_fsi_base_solver.py

```diff
@@ -22,6 +22,16 @@ class TrilinosPartitionedFSIBaseSolver(partitioned_fsi_base_solver.PartitionedFS
     def __init__(self, model, project_parameters):
         super(TrilinosPartitionedFSIBaseSolver, self).__init__(model, project_parameters)
 
+    @classmethod
+    def GetDefaultSettings(cls):
+        """This function returns the default-settings used by this class
+        """
+        this_defaults = KratosMultiphysics.Parameters("""{
+            "parallel_type": "MPI"
+        }""")
+        this_defaults.AddMissingParameters(super(PartitionedFSIDirichletNeumannSolver, cls).GetDefaultSettings())
```

Comment:
```suggestion
        this_defaults.AddMissingParameters(super(TrilinosPartitionedFSIBaseSolver, cls).GetDefaultSettings())
```

Reply:
I might be wrong, not sure, just wanted to point it out
if intended then an explanatory comment would help

Reply:
You're right, copy & paste error. Thanks for spotting.